script.js

```diff
@@ -0,0 +1,59 @@
+/* eslint-disable no-unused-vars */
+
+const form = document.querySelector('form');
+const bookInfo = document.querySelector('.all-books');
+const title = document.querySelector('#input-title');
+const author = document.querySelector('#input-author');
+
+let bookArr = [];
+
+function bookDisplay() {
+  bookInfo.innerHTML = '';
+  for (let i = 0; i < bookArr.length; i += 1) {
+    const oneBook = document.createElement('div');
+    oneBook.className = 'book';
+    oneBook.innerHTML = `
+      <p class="title">Title: ${bookArr[i].title}</p>
+      <p class="author">Author: ${bookArr[i].author}</p>
+      <button class="remove" onclick="removeBook(${i})">Remove</button>
+      <hr />
+    `;
+    bookInfo.appendChild(oneBook);
+    title.value = '';
+    author.value = '';
+  }
+}
+
+function addBook() {
+  const eachBook = {};
+  // const eachBook = new Object();
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉